src/interpolation_caches.jl

```diff
@@ -22,18 +27,22 @@ struct LinearInterpolation{uType, tType, IType, pType, T} <: AbstractInterpolati
     extrapolate::Bool
     idx_prev::Base.RefValue{Int}
     cache_parameters::Bool
-    function LinearInterpolation(u, t, I, p, extrapolate, cache_parameters)
+    use_linear_lookup::Bool
```

Comment:
```suggestion
    linear_lookup::Bool
```

It would be more consistent with how `linear_lookup` is computed in the constructor.